sklearn/base.py

```diff
@@ -359,10 +359,32 @@ def score(self, X, y, sample_weight=None):
         -------
         score : float
             R^2 of self.predict(X) wrt. y.
+
+        Notes
+        -----
+        The default value of ``multioutput`` in the ``score`` method of will
+        change from ``variance_weighted`` to ``uniform_average`` in 0.23 to
```

Comment:
my preference is for quotes rather than code when we are talking about string constants